app/apply.py

```diff
@@ -0,0 +1,24 @@
+def apply_everything(name_of_knight: str, knights_config: dict) -> dict:
```

Comment:
`everything` is not really descriptive. Change it. You can create different functions, classes to make a logic much more clear

Reply:
Also change a module name